### ISSUE TITLE
New Learning Pathways installation in English is provisioned with navigation links and content in Dutch

### ISSUE THREAD

**Issue by bryanhart73** [SharePoint/sp-provisioning-service#542]
**Describe the issue**
Created a new Learning Pathways installation in a tenant that has never had it before. Followed the instructions at https://docs.microsoft.com/en-us/office365/customlearning/custom_provision . Using this resulted in an English site collection but the navigation links were in a mix of English and Dutch.

**Expected behavior**
An English site should have English navigation links.

**Screenshots**

**Details (please complete the following information):**
- PnP Correlation ID: N/A
- Reference Template: https://lookbook.microsoft.com/details/3df8bd55-b872-4c9d-88e3-6b2f05344239
- Primary language of the target tenant: EN
- Are you using a Tenant Global Admin account to provision the template: Yes

**Additional context**
Originally reported to https://github.com/pnp/custom-learning-office-365/issues/539

**Comment by gregvolz** [comment]
I just did this as well and followed these steps - [Scenario 2](https://docs.microsoft.com/en-us/office365/customlearning/custom_setupoptions#scenario-2-we-installed-learning-pathways-but-arent-currently-using-it-andor-havent-made-any-customization-to-the-site-or-playlists)

It seems as if the step to "[Ensure the Tenant Admin account doesn't have a language selected](https://docs.microsoft.com/en-us/office365/customlearning/custom_provision#ensure-the-tenant-admin-account-doesnt-have-a-language-selected)" may be causing the issue.

I went back into my Tenant Admin account and selected English as my default language and then followed the instructions again to provision a new Learning Pathways site /M365LP3 vs. my /M365LP2 (with Dutch header links and English content) and now have an all English site.
